test/ao/sparsity/test_data_sparsifier.py

```diff
@@ -230,3 +232,209 @@ def test_nn_embeddings(self):
         emb_test = TestBaseDataSparsiferType(data_list=data_list, defaults=defaults,
                                              data_with_config=data_with_config)
         emb_test.run_tests()
+
+
+class TestNormDataSparsifierType(TestBaseDataSparsiferType):
```

Comment:
Similar to https://github.com/pytorch/pytorch/pull/79252/files#r897301980 I'm not understanding the name here, feels like it should be something like TestBaseDataSparsifier(Generator/Helper/Runner) or something like that and it should have a comment similar to the comment for the base class describing this.

Reply:
Renamed it to TestDataNormSparsifiersRunner and added comments